src/Storages/MergeTree/MergeTreeMetadataCache.cpp

```diff
@@ -0,0 +1,70 @@
+#include "MergeTreeMetadataCache.h"
+
+#if USE_ROCKSDB
+#include <Common/ProfileEvents.h>
+#include <base/logger_useful.h>
+
+namespace ProfileEvents
+{
+    extern const Event MergeTreeMetadataCachePut;
+    extern const Event MergeTreeMetadataCacheGet;
+    extern const Event MergeTreeMetadataCacheDelete;
+    extern const Event MergeTreeMetadataCacheSeek;
+}
+
+namespace DB
+{
+MergeTreeMetadataCache::Status MergeTreeMetadataCache::put(const String & key, const String & value)
+{
+    auto options = rocksdb::WriteOptions();
```

Comment:
Options is not safe, at least [`WriteOptions::sync`](https://github.com/facebook/rocksdb/blob/677d2b4a8f8fd19d0c39a9ee8f648742e610688d/include/rocksdb/options.h#L1631) should be set (disableWAL is set by default but worth setting it explicitly I guess)

Or, at least, it should be done under some option (like fsync_after_insert)

Reply:
done.